config/doc.go

```diff
@@ -0,0 +1,4 @@
+//go:generate go-composite-schema --required config global-config.yml config.go
+//
+// Package config stores the config for the taskcluster-worker
+package config
```

Comment:
I propose this lives in the `/worker/` folder/package... currently named taskmgr